### PR TITLE
chore: add references to ipfs-gateway.md

### PR DESCRIPTION
**Pull request by lidel**
- [x] Concept page should link to relevant sections of "Address IPFS on the web" tutorial.
- [x] Remove link to article which anti-pattern
- [x] CI green

### CODE REVIEW

File: docs/concepts/ipfs-gateway.md [hunk 1]
```diff
@@ -66,19 +64,12 @@ _Private gateways_ are configured to limit access to requests from specific doma
 
 They are frequently, but not exclusively, used behind firewalls. Running [IPFS Desktop](https://github.com/ipfs-shipyard/ipfs-desktop#ipfs-desktop) or another form of IPFS node triggers connection attempts to other IPFS peers. Private network administrators may treat such connection attempts as potential security vulnerabilities. Private IPFS gateway servers located inside the private network and running a trusted code base provide an alternative architecture for read/write access to externally-hosted IPFS content.
 
-This [tutorial configuring an IPFS gateway on a Google Cloud platform](https://blog.stacktical.com/ipfs/gateway/dapp/2019/09/21/ipfs-server-google-cloud-platform.html) includes information on constraining access for a private gateway setup.
```
Comment (lidel) [inline docs/concepts/ipfs-gateway.md:69]: :thought_balloon: Removed due to CI getting 403 + content promoting antipattern, exposes subset of RPC port to internet, anyone can upload.

File: docs/concepts/ipfs-gateway.md [hunk 2]
```diff
@@ -118,11 +109,21 @@ https://{gateway URL}/ipfs/{content ID}/{optional path to resource}
 
 Path-resolving gateways, however, violate the [same-origin policy](https://en.wikipedia.org/wiki/Same-origin_policy) that protects one website from improperly accessing session data of another website.
 
+:::warning
+This type of gateway does not provide origin isolation and should not be used for hosting web apps.
+
+Learn more at [Address IPFS on the web: Path Gateway](../how-to/address-ipfs-on-web.md#path-gateway)
```
Comment (lidel) [inline docs/concepts/ipfs-gateway.md:115]:
```suggestion
Learn more at [Address IPFS on the web: Path Gateway](../how-to/address-ipfs-on-web.md#path-gateway) and [Path Gateway Specification](https://specs.ipfs.tech/http-gateways/path-gateway/).
```

File: docs/concepts/ipfs-gateway.md [hunk 3]
```diff
@@ -118,11 +109,21 @@ https://{gateway URL}/ipfs/{content ID}/{optional path to resource}
...
+
 #### Subdomain
 
 Subdomain resolution style maintains compliance with the [single-origin policy](https://en.wikipedia.org/wiki/Same-origin_policy). The canonical form of access, `https://{CID}.ipfs.{gatewayURL}/{optional path to resource}`, causes the browser to interpret each returned file as being from a different origin.
 
-Subdomain resolution support began with [Kubo](https://github.com/ipfs/kubo) release `0.5.0`.
+::: callout
+This type of gateway does provide origin isolation and should be used for hosting web apps.
+
+Learn more at [Address IPFS on the web: Subdomain Gateway](../how-to/address-ipfs-on-web.md#subdomain-gateway)
```
Comment (lidel) [inline docs/concepts/ipfs-gateway.md:125]:
```suggestion
Learn more at [Address IPFS on the web: Subdomain Gateway](../how-to/address-ipfs-on-web.md#subdomain-gateway) and [Subdomain Gateway Specification](https://specs.ipfs.tech/http-gateways/subdomain-gateway/).
```

File: docs/concepts/ipfs-gateway.md [hunk 4]
```diff
@@ -145,6 +146,10 @@ DNSLink resolution occurs when the gateway recognizes an IPNS identifier contain
 4. The gateway employs DNSLink resolution to return the current content version from IPFS.
 5. The browser does not perceive the gateway as the origin of the content and therefore enforces the single-origin policy to protect `example.com`.
 
+::: callout
+Learn more at [Address IPFS on the web: DNSLink Gateway](../how-to/address-ipfs-on-web.md#dnslink-gateway)
```
Comment (lidel) [inline docs/concepts/ipfs-gateway.md:150]:
```suggestion
Learn more at [Address IPFS on the web: DNSLink Gateway](../how-to/address-ipfs-on-web.md#dnslink-gateway) and [DNSLink Gateway Specification](https://specs.ipfs.tech/http-gateways/dnslink-gateway/).
```

**Review by 2color — COMMENTED** [review]
LGTM

**Pull request merged**